src/xargs/mod.rs

```diff
@@ -818,6 +847,25 @@ fn do_xargs(args: &[&str]) -> Result<CommandResult, XargsError> {
                 .help("Be verbose")
                 .action(ArgAction::SetTrue),
         )
+        .arg(
+            Arg::new(options::REPLACE)
+                .long(options::REPLACE)
+                .short('i')
+                .num_args(0..=1)
+                .require_equals(true)
+                .value_parser(clap::value_parser!(String))
```

Comment:
The help text mentions `R`.
```suggestion
                .value_parser(clap::value_parser!(String))
                .value_name("R")
```